pvlib/pvsystem.py

```diff
@@ -2878,14 +2878,34 @@ def scale_voltage_current_power(data, voltage=1, current=1):
 
 @renamed_kwarg_warning(
     "0.13.0", "g_poa_effective", "effective_irradiance")
-def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
+def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.,
+               k=None, cap_adjustment=False):
     r"""
-    Implements NREL's PVWatts DC power model. The PVWatts DC model [1]_ is:
+    Implements NREL's PVWatts (Version 5) DC power model. The PVWatts Version
+    5 DC model [1]_ is:
```

Comment:
I suggest (an imperative) single sentence for the one line description (single sentence helps with the sphinx autosummary for functions) 
Also, let's follow new(-ish?) pvlib style: single-line description -> parameter description -> notes section.
So I suggest moving everything "The PVWatts Version..." onwards to _after_ the parameter description. [Example](https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.spectrum.average_photon_energy.html#pvlib-spectrum-average-photon-energy)

```suggestion
    Implement NREL's PVWatts (Version 5) DC power model. The PVWatts Version
    5 DC model [1]_ is:
```